app/assets/config/locale/en-US.json

```diff
@@ -165,23 +165,23 @@
     "indexQueryPrompt_prefix": "In the ",
     "indexQueryPrompt_suffix": " space, no tag indexes are found, so query by index is not available.",
     "indexQueryPrompt2": "Please create indexes on the tags. Here is an example:",
-    "runCodeInConsole": "Run code in the console",
+    "runCodeInConsole": "Execute queries in the console",
     "indexLink": "For more information about index, see its ",
     "documentIntroduction": "documentation",
     "selectIndex": "Select a Index",
     "paramFilter": "Use Index",
-    "relationship": "Filter Logic",
+    "relationship": "Logical Operator",
     "operationConfirm": "This Delete operation will clear the following fields. Are you sure you want to continue the operation?",
     "quiry": "Query",
     "customQueryDescription": "Enter the statements in the console. When the results are returned, click the Open in Explore button as shown in the preceding figure to explore the graph.",
     "openInConsole": "Go to Console",
-    "insertMethodSelect": "Some data on the board. Please select how to insert the new queried results",
-    "incrementalInsertion": "Incremental Insertion",
-    "insertAfterClear": "Insert After Clear",
+    "insertMethodSelect": "How do you like to render the new data with the existing graph view, incrementally or wipe the graph view first?",
```

Comment:
"incrementally or wipe the graph view first" sounds a little bit strange here.

do you want to say render incrementally or xxx?